components/ui/components/design/atoms/inputs/Checkbox.tsx

```diff
@@ -0,0 +1,161 @@
+import { JSX, Show, splitProps } from "solid-js";
+import { styled } from "solid-styled-components";
+import { BiRegularCheck } from 'solid-icons/bi'
+
+const Base = styled("label")`
+  gap: 10px;
+  padding: 4px;
+  display: flex;
+  cursor: pointer;
+  user-select: none;
+  align-items: center;
+  border-radius: ${({ theme }) => theme.borderRadius["md"]};
+  transition: 0.1s ease background-color;
+  input {
+    display: none;
+  }
+  &:hover {
+    background: ${({ theme }) => theme.colours["background-100"]};
+    .playSound {
+      visibility: visible;
+      opacity: 1;
+    }
+    .check {
+      visibility: visible;
+      opacity: 1;
+    }
+  }
+  &[disabled] {
+    opacity: 0.8;
+    cursor: not-allowed;
+  }
+  //TODO: When value is checked, allow me to add .hover { .checkmark { border-color:  var(--tertiary-foreground);} }
```

Comment:
This comment doesn't need to be included.
Note: may cause issues with goober, comments must be `/* */`.